config/config.py

```diff
@@ -105,7 +105,7 @@ def build_config(
     )
     # Tau base selection:
     configuration.add_config_parameters(
-        "global",
+        ["global", "et", "mt", "tt"],
```

Comment:
keep the selection and the definition of paths etc seperate here, so pt and eta cut for global only and only setting the correctionlib parameters for all 4 scopes